docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2092,27 +2298,36 @@ function addMedia() {
     preferLiveVideoElm.value = 'Enable';
   }
 
-  // addMedia using the default RemoteMediaManagerConfig
-  meeting.addMedia({
+  const addMediaOptions = {
     localTracks: {
       microphone: localMedia.microphoneTrack,
       camera: localMedia.cameraTrack,
     },
     ...getMediaSettings()
+  };
+
+  if (localMedia.screenShare) {
+    addMediaOptions.localTracks.screenShare = {
+      video: localMedia.screenShare.video,
     }
-  ).then(() => {
-    // we need to check shareStatus, because may have missed the 'meeting:startedSharingRemote' event
-    // if someone started sharing before our page was loaded,
-    // or we didn't act on that event if the user clicked "add media" while being in the lobby
-    if (isMultistream && meeting.shareStatus === 'remote_share_active') {
-      forceScreenShareViewLayout(meeting);
-    }
+  }
 
-    console.log('MeetingStreams#addMedia() :: successfully added media!');
-  }).catch((error) => {
-    console.log('MeetingStreams#addMedia() :: Error adding media!');
-    console.error(error);
-  });
+  // addMedia using the default RemoteMediaManagerConfig
+  meeting.addMedia(addMediaOptions)
+    .then(() => {
```

Comment:
We can use async-await here as well

Reply:
I would recommend to do these type of refactoring changes in a separate task